libsel4camkes/include/camkes/virtqueue.h

```diff
@@ -54,6 +54,8 @@ extern camkes_virtqueue_channel_t camkes_virtqueue_channels[MAX_CAMKES_VIRTQUEUE
 /* The number of virtqueue channels registered to a component */
 extern int num_registered_virtqueue_channels;
 
+camkes_virtqueue_channel_t *get_virtqueue_channel(virtqueue_role_t role, unsigned int camkes_virtqueue_id);
```

Comment:
The other functions have some comment with a description, would be nice to have this here then also not that this is public. Might say explicitly that channels without a buffer are never returned (unless this mean something is broken anyway, so it's not supposed to happen?)

Reply:
Is this ok to be resolved?

Reply:
looks good, thanks.